subprojects/core/src/main/java/org/gradle/api/services/internal/BuildServiceDetails.java

```diff
@@ -46,7 +47,7 @@ public BuildServiceDetails(BuildIdentifier buildIdentifier, String name, Class<T
         this.name = name;
         this.implementationType = implementationType;
         this.parameters = null;
-        this.maxUsages = -1;
+        this.maxUsages = DEFAULT_MAX_USAGES;
```

Comment:
💅 There is another instance of `-1` with the same meaning in the overloaded constructor above (line 41).

Reply:
Relied blindly on the refactor to include all occurrences...